test/acceptance/profile_test.rb

```diff
@@ -29,8 +29,10 @@ def test_profile_shows_progress
 
   def test_display_team_invites_only_in_users_own_profile
     new_user = create_user(username: 'new_user', github_id: 456)
-    attributes = { slug: 'some-team', name: 'Some Name', usernames: 'new_user' }
-    Team.by(@user).defined_with(attributes, @user).save!
```

Comment:
There's probably a good reason, but why does this need to change?

It seems like creating a team with existing members is a useful thing to be able to do in one shot.
This would probably require different changes to `lib/exercism/team.rb` but won't require changing everything else.